tests/trinity/core/configuration/test_trinity_config_object.py

```diff
@@ -90,3 +91,12 @@ def test_trinity_config_explictely_provided_nodekey(nodekey_bytes, as_bytes):
     )
 
     assert trinity_config.nodekey.to_bytes() == nodekey_bytes
+
+
+def test_trinity_config_sub_configs():
+    trinity_config = TrinityConfig(network_id=1)
+    trinity_config.initialize_app_configs(None, (BeaconAppConfig,))
+
+    assert trinity_config.has_app_config(BeaconAppConfig)
```

Comment:
Curious what others think about this.  I've often done `assert thing is True` when I am actually testing a boolean value to ensure that some other "truthy" value cannot slip through.